app/content/carto-user-manual/settings/understanding-your-organization-quotas.md

```diff
@@ -0,0 +1,77 @@
+
+{{% bannerNote title="Example"%}}
+If I already have 3 out of 3 editors available, I won't be able to invite new editors or promote viewers to editors.
+{{%/ bannerNote %}}
+
+--- 
+
+#### Viewer users
+
+A viewer is a user that can only view the objects (maps, applications...) produced in the organization. They can't create or edit anything in the organization... This is a hard quota, and will be enforced whenever users try to signup or whenever you try to add new viewers through invitations or roles. For more information on how to manage users, check [Managing User Roles](/carto-user-manual/settings/managing-user-roles/).
+
+{{% bannerNote title="Example"%}}
+If I already have 50 out of 50 editors available, I won't be able to invite viewers or demote editors to viewers. Also, users that try to join or request to join the organization will be blocked.
+{{%/ bannerNote %}}
+
+--- 
+
+#### Total maps
+
+All maps (public, private or shared) from all users count towards this quota. A good practice to manage this quota is to have all the users delete unused maps. For more information on how to manage maps, check the [Maps documentation](/carto-user-manual/maps/introduction/).
+
+{{% bannerNote title="Example"%}}
+If I already have 100 out of 100 maps, I won't be able to create new maps or duplicate any existing maps. I will still be able to modify the sharing settings of existing maps.
+{{%/ bannerNote %}}
+
+--- 
+
+#### Public maps
+
+Only maps that are shared as "Public" count towards this quota. For more information on sharing maps, check [Publishing and Sharing Maps](/carto-user-manual/maps/publishing-and-sharing-maps/).
+
+Maps shared only with the organization don't count towards this quota.
+
+{{% bannerNote title="Example"%}}
+If I already have 10 out of 10 public maps, I won't be able to share new maps as "Public" until I unpublish a existing map or increase the quota.
+{{%/ bannerNote %}}
+
+--- 
+
+#### Applications
+
+Applications are the way to create custom developments with CARTO. They are created under the Developers section of the Workspace. Registered apps (the ones you register in Settings) don't count towards this quota. Learn more in the [Applications documentation](/carto-user-manual/applications/accessing-applications/). 
+
+{{% bannerNote title="Example"%}}
+If I already have 5 out of 5 applications, I won't be able to create new applications in the "Developers" section.
+{{%/ bannerNote %}}
+
+--- 
+
+#### Connections
+
+The way to get, analyze and visualize data coming from your cloud data warehouse is through a **connection**. You can create a connection with Google BigQuery, Snowflake, Amazon Redshift, Databricks or PostgreSQL. Learn more about [Creating and Managing connections](/carto-user-manual/connections/introduction/).
+
```

Comment:
We could redirect directly to "creating a connection" section, where the user can see all the subsections depends on the provider.

Reply:
Hmmm, I think it's too detailed, I like the intro better because from there it redirects to the different sections (create/edit/delete...) 💭 Is that okay?